tests/v2/actions/observability/stackstate.go

```diff
@@ -0,0 +1,212 @@
+package observability
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	v1 "github.com/rancher/rancher/pkg/apis/catalog.cattle.io/v1"
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/defaults"
+	"github.com/rancher/shepherd/pkg/wait"
+	log "github.com/sirupsen/logrus"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	StackstateName         = "stackstate"
```

Comment:
if you're going to make these public, I think there should only be one. you have it public here and in actions/charts

Reply:
this constant is only limited to the ui plugins but not the charts. I donot have this in actions/charts. I'm adding this constant here to avoid importing charts just for these constants.